passkeeper/tests/units/test_passkeeper.py

```diff
@@ -64,16 +64,17 @@ def test_init_dir(self, mock_create_dir, mock_encrypt, mock_cleanup):
         mock_cleanup.assert_called_once_with()
 
 
-    @patch('passkeeper.Passkeeper._remove_old_encrypted_files')
+    @patch('passkeeper.Passkeeper.remove_old_encrypted_files')
```

Comment:
Same as comment below, cleanup don't call remove anymore, no need to mock and test it here.